sdk/ml/azure-ai-ml/azure/ai/ml/distillation/_distillation.py

```diff
@@ -0,0 +1,111 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Entrypoint for creating Distillation task."""
+from typing import Any, Dict, Optional, Union
+
+from azure.ai.ml._utils._experimental import experimental
+from azure.ai.ml.constants import DataGenerationType
+from azure.ai.ml.constants._common import AssetTypes
+from azure.ai.ml.entities._inputs_outputs import Input
+from azure.ai.ml.entities._job.distillation.distillation_job import DistillationJob
+from azure.ai.ml.entities._job.distillation.prompt_settings import PromptSettings
+from azure.ai.ml.entities._job.distillation.teacher_model_settings import TeacherModelSettings
+from azure.ai.ml.entities._job.resource_configuration import ResourceConfiguration
```

Comment:
should we keep this as JobResourceConfiguration to be aligned with your _schema/_distillation resources?
resources = NestedField(JobResourceConfigurationSchema)

Reply:
Changed the schema type to instead use ResourceConfigurationSchema as that aligns more with your schema in azureml_run_configuration. Also, ResourceConfiguration is what is used in the Distillation.md as well